packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -215,9 +205,18 @@ class DropDownButton<T> extends StatelessComponent {
   void _showDropDown(BuildContext context, int selectedIndex, GlobalKey indexedStackKey) {
     final RenderBox renderBox = indexedStackKey.currentContext.findRenderObject();
     final Rect rect = renderBox.localToGlobal(Point.origin) & renderBox.size;
+    final NavigatorState navigator = Navigator.of(context);
```

Comment:
yeah, should be possible. Or have it keep track of the outline. Assign the
bug to me, I'll take a look.

On Tue, Dec 1, 2015, 09:42 Adam Barth <notifications@github.com> wrote:

> In packages/flutter/lib/src/material/dropdown.dart
> <https://github.com/flutter/flutter/pull/644#discussion_r46311034>:
>
> > @@ -215,9 +205,18 @@ class DropDownButton<T> extends StatelessComponent {
> >    void _showDropDown(BuildContext context, int selectedIndex, GlobalKey indexedStackKey) {
> >      final RenderBox renderBox = indexedStackKey.currentContext.findRenderObject();
> >      final Rect rect = renderBox.localToGlobal(Point.origin) & renderBox.size;
> > +    final NavigatorState navigator = Navigator.of(context);
>
> Maybe we can change the route machinery to clear the navigator later.
> @Hixie <https://github.com/Hixie>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flutter/flutter/pull/644/files#r46311034>.
>